css/index.css

```
@@ -1 +1,178 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Inter", sans-serif;
+  display: flex;
+}
 
+@font-face {
+  font-family:'Nexa Extra Light';
+  font-family:'Nexa Heavy';
+  src: url('../fonts/Nexa-ExtraLight.ttf') format('truetype');
+  src: url('../fonts/Nexa-Heavy.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+.login-information__container{
+  max-width: 1440px;
+  margin: 0 auto;
+  margin-top: 100px;
+  display: flex;
+  background-color: var(--color-white);
+}
+
+.Earnings{
+  flex-basis: 604px;
+}
+
+.carousel__Earnings{
+  list-style-type: none;
+  padding-left: 0px;
+  margin-top: 0px;
+}
+
+.Earnings__left-side{
+  background-color: var(--color-purple);
+  color: var(--color-white);
+  padding: 30px 59px 75px 24px;
+  display: flex;
+  flex-direction: column;
+}
+
+.Earnings__image{
+  width: 210px;
+  height: 70px;
+}
+
+.Earnings__found{
+  margin-top: 65px;
+  margin-left: 32px
+}
+
+.Earnings__info{
+  display: flex;
+  background-color: var(--color-white);
+  box-shadow: 0px 3.77px 49.95px 0px #0000001A;
+  border-radius: 19px;
+  width: 480px;
+  height: 257px;
+  align-items: center;
+  justify-content: center;
+  margin-left: 32px;
+  margin-top: -23px;
+}
+
+.Earnings__details{
+  margin-left: 42px;
+}
+
+.Earnings__earn{
+  color: var(--color-strong-gray);
+  font-size: var(--font-size-xlarge);
+  font-weight: var(--font-weight-boldM);
+  line-height: var(--line-heightx-large);
+  font-family:'Nexa Heavy';
+  margin: 0px;
+  margin-top: -20px;
+  padding-bottom: 7px;
+}
+
+.Earnings__total{
+  font-family:'Nexa Heavy';
+  font-size: 17px;
+  font-weight: var(--font-weigh);
+  line-height: var(--line-height-small);
+  color: var(--color-soft-gray);
+}
+
+.Earnings__price{
+  font-family:'Nexa Heavy';
+  font-size: var(--font-size-xlarge);
+  font-weight: var(--font-weight-boldM);
+  line-height: var(--line-heightx-large);
+  color: var(--color-strong-gray);
+  margin: 0px;
+  padding-top: 10px;
+  padding-bottom: 28px;
+}
+
+.Earnings__profit{
+  font-family:'Nexa Heavy';
```

Tambien dice el diseño que aca es `regular`
```suggestion
  font-family:'Nexa';
```